nncf/api/compression.py

```diff
@@ -11,8 +11,16 @@
  limitations under the License.
 """
 
-from abc import ABC, abstractmethod
-from typing import Any, Dict, Optional, TypeVar, List, Tuple
+from abc import ABC
+from abc import abstractmethod
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+from typing import Tuple
+from typing import TypeVar
+from typing import Union
```

Comment:
I think it's preferable to use one import for all typing imports https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#22-imports

Reply:
@alexsu52 @vshampor 
IDE whether splits or joins imports. Any known mechanism to automatically follow the guidelines? 
![image](https://user-images.githubusercontent.com/4014476/120982497-71452e80-c781-11eb-839e-bbf329300e12.png)

Reply:
@ljaljushkin I don't know of such a mechanism in the IDE.